package.json

```diff
@@ -43,7 +43,7 @@
   },
   "peerDependencies": {
     "@sanity/client": "^2.11.0",
-    "next": "^11.0.0",
+    "next": "^11.0.0 ^12.0.0",
```

Comment:
Providing multiple package versions is not supported, could you change this to be just `^12.0.0`?

Reply:
Sorry, my fault, I forgot to add the logical or `||` for the semver range 🤦 
Only using `^12.0.0` would make it a breaking change for this package and is not particular needed here.

PR is updated to support both.